staging/gatekeeper/templates/pre-delete-cleanup-job.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "gatekeeper.fullname" . }}-cleanup
+  labels:
+    {{- include "gatekeeper.commonLabels" . | nindent 4 }}
+  annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+spec:
+  template:
+    spec:
+      serviceAccount: {{ template "gatekeeper.serviceAccountName" . }}
+      {{- if or (.Values.securityContext.runAsUser) (.Values.securityContext.fsGroup) }}
+      securityContext:
+        {{- if .Values.securityContext.runAsUser }}
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+        {{- end }}
+        {{- if .Values.securityContext.fsGroup }}
+        fsGroup: {{ .Values.securityContext.fsGroup }}
+        {{- end }}
+      {{- end }}
+      containers:
+      - name: cleanup
+        image: bitnami/kubectl:1.15
```

Comment:
It won't be bad to have this version in values

Reply:
Done.